### ISSUE TITLE
Call to undefined function Spatie\EventProjector\Models\now()

### ISSUE THREAD

**Issue by dimatock** [spatie/laravel-event-projector#108]
Lumen 5.6

vendor/spatie/laravel-event-projector/src/Models/StoredEvent.php:31

```
public static function createForEvent(ShouldBeStored $event): StoredEvent
{
        $storedEvent = new static();
        $storedEvent->event_class = get_class($event);
        $storedEvent->attributes['event_properties'] = app(EventSerializer::class)->serialize(clone $event);
        $storedEvent->meta_data = [];
        $storedEvent->created_at = now(); 

        $storedEvent->save();

        return $storedEvent;
}
```

**Comment by dimatock** [comment]
https://github.com/spatie/laravel-event-projector/pull/109

**Comment by freekmurze** [comment]
Fixed via #109
